src/ui/popup.js

```diff
@@ -539,46 +582,22 @@ export default class Popup extends Evented {
             this._lngLat = smartWrap(this._lngLat, this._pos, this._map.transform);
         }
 
-        if (this._trackPointer && !cursor) return;
+        if (!this._trackPointer || cursor) {
```

Comment:
Nit: Is there a reason for inverting the conditional? Early return can be nice to de-nest code when otherwise equivalent. Nothing blocking here for me though.

Reply:
My thinking was to remove the early return allowing `this. _updateClassList()` to be called in any condition.

Reply:
Yup it was redundant! (Oops) Fixed in latest commit.